content/en/synthetics/metrics.md

```diff
@@ -23,9 +23,13 @@ Metrics starting with:
     * `synthetics.http.*` come from your API [HTTP tests][2]
     * `synthetics.ssl.*` come from your API [SSL tests][3]
     * `synthetics.dns.*` come from your API [DNS tests][4]
-    * `synthetics.tcp.*` come from your API [TCP tests][5]
-    * `synthetics.icmp.*` come from your API [ICMP tests][6]
-* `synthetics.browser.*` come from your [browser tests][7]
+    * `synthetics.websocket.*` come from your API [WebSocket tests][5]
+    * `synthetics.tcp.*` come from your API [TCP tests][6]
+    * `synthetics.udp.*` come from your API [UDP tests][7]
+    * `synthetics.icmp.*` come from your API [ICMP tests][8]
+* `synthetics.multi.response.time` come from your [multistep API tests][9]
```

Comment:
```suggestion
* `synthetics.multi.*` come from your [multistep API tests][9]
```

Just incase we add more multistep metrics in the future

Reply:
Good idea!